src/gui/utils.cpp

```diff
@@ -68,7 +68,10 @@ qreal Utils::Gui::screenScalingFactor(const QWidget *widget)
 #ifdef Q_OS_WIN
     const int screenNumber = qApp->desktop()->screenNumber(widget);
     const QScreen *screen = QApplication::screens()[screenNumber];
-    return (screen->logicalDotsPerInch() / screen->physicalDotsPerInch());
+    // Workaround for QScreen::physicalDotsPerInch() that could return
+    // values that are smaller than the normal 96 DPI on Windows
```

Comment:
Is it known bug?
What can cause it?

Reply:
>Is it known bug?

No.

>What can cause it?

No idea, the workaround here is merely deduced from the issue report. Or could be related to monitor EDID.